src/test/java/com/google/cloud/tools/managedcloudsdk/install/TarGzExtractorProviderTest.java

```diff
@@ -68,8 +67,9 @@ public void testZipSlipVulnerability_windows() throws URISyntaxException {
       tarGzExtractorProvider.extract(testArchive, extractionRoot, mockProgressListener);
       Assert.fail("IOException expected");
     } catch (IOException ex) {
-      Assert.assertThat(
-          ex.getMessage(), Matchers.startsWith("Blocked unzipping files outside destination: "));
+      String message = ex.getMessage();
+      Assert.assertNotNull(message);
```

Comment:
Seems like this should remain assertThat (with new package) with containsString, would remove need for null check.

Reply:
done